### PR TITLE
feat(RHTAPREL-804): update internal service reference for migration

### PR DESCRIPTION
**Pull request by happybhati**
-  This commit updates the `internal-service` reference
as result of migration to `konflux-ci` from `redhat-appstudio`
 more details parent EPIC:
 https://issues.redhat.com/browse/RHTAPREL-800

### CODE REVIEW

**Comment by johnbieren** [comment]
This PR is proof CODEOWNERS is not working in this repo

**Review by johnbieren — CHANGES_REQUESTED** [review]

File: CODEOWNERS
Comment (johnbieren) [inline CODEOWNERS:1]: I think the content was correct, but it should be in the `.github` dir 
See https://github.com/konflux-ci/release-service-utils/blob/main/.github/CODEOWNERS

Comment (johnbieren) [reply CODEOWNERS:1]: If you are fixing this in this PR, can you make it a separate commit?

Comment (happybhati) [reply CODEOWNERS:1]: Sure, on it.

**Comment by happybhati** [comment]
> The second commit has the move of the CODEOWNERS, but not the update. Nit pick, but it would be ideal to have the move and update part of that commit, and no CODEOWNERS change in the first commit 🙂 If you end up in git hell, we can just merge

Fixed 😉

**Review by johnbieren — APPROVED** [review]

**Pull request merged**